### PR TITLE
[Databricks]az databricks workspace create: Add new parameters --publ…

### PR DESCRIPTION
**Pull request by ZengTaoxu**
…ic-network-access and --required-nsg-rules

update databricks sdk version and add new parameters

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Databricks

File: src/databricks/azext_databricks/_params.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
     get_three_state_flag
 )
 
+from .vendored_sdks.databricks.models import PublicNetworkAccess, RequiredNsgRules
```
Comment (kairu-ms) [inline src/databricks/azext_databricks/_params.py:16]: Move this line into load_arguments function

**Comment by kairu-ms** [comment]
Please update extension version in https://github.com/Azure/azure-cli-extensions/blob/71cf01ee6874657354daa43c0de8b67b8b82ff9b/src/databricks/setup.py#L19

**Comment by kairu-ms** [comment]
Please add release history in https://github.com/Azure/azure-cli-extensions/blob/main/src/databricks/HISTORY.rst

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**